app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(10, 3)) == 10
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    assert min(split_integer(9, 3)) == max(split_integer(9, 3))
```

Comment:
you should write result, not result from a function
ex:
`assert split_integer(200, 4) == [50, 50, 50, 50]`